lib/active_snapshot/models/snapshot_item.rb

```diff
@@ -12,22 +12,21 @@ class SnapshotItem < ActiveRecord::Base
     validates :snapshot_id, presence: true
     validates :item_id, presence: true, uniqueness: { scope: [:snapshot_id, :item_type] }
     validates :item_type, presence: true
+    validates :object, presence: true
 
     def object
       return @object if @object
 
       if ActiveSnapshot.config.storage_method_json?
-        @object = JSON.parse(self[:object])
+        @object = self[:object] ? JSON.parse(self[:object]) : {}
```

Comment:
Instead of doing this empty object fallback on every line Maybe we can do `@object ||= {}` after the if else storage method statements